bin/combinetf2_plot_hists.py

```diff
@@ -747,21 +791,28 @@ def make_plot(
                         hdiff = hh.scaleHist(hdiff, scaleVariation[i])
                         hd = hh.addHists(hdiff, h_inclusive)
 
+                if diff:
```

Comment:
what are these changes about?  add something to the PR description about that?

Reply:
Updated description